search-sentiments-map/routes/search.js

```diff
@@ -68,19 +71,21 @@ async function retrieveSearchResultFromDatastore(topic) {
 }
 
 /** 
- * Updates daily search results (accumulates by day) in the Datastore.
+ * Updates daily search results (accumulated by day) in the Datastore.
  */
 function updateSearchResults() {
   retrieveGlobalTrends().then(async trends => {
-    for (let i = 0; i < trends.length; i++) { // When testing, make i < 1 trend to test one.
+    // When testing, make i < 1 to test for only one trend and comment out
+    // await new Promise line to avoid 1 minute pauses.
```

Comment:
```suggestion
    // When testing, use i < 1 to test for only one trend, and comment out
    // `await new Promise` line to avoid 1 minute pauses.
```